conda_libmamba_solver/shards_subset.py

```diff
@@ -127,22 +162,22 @@ def outgoing(self, node: Node):
         for n in self.neighbors(node):
             yield n, 1
 
-    def shortest(self, start_packages):
+    def shortest_dijkstra(self, root_packages):
```

Comment:
If it was proper un-buggy Dijkstra's, it should be breadth-first. "Dijkstra's algorithm finds the shortest path from a given source node to every other node." (Wikipedia)
It doesn't really matter what we call it.
I like continuing to compute the distance to each node for analysis purposes.